packages/office-ui-fabric-react/src/utilities/selection/examples/Selection.Basic.Example.tsx

```diff
@@ -81,160 +81,128 @@ const SelectionItemExample: React.FunctionComponent<ISelectionItemExampleProps>
-      </div>
-    );
-  }
-
-  private _alertItem = (item: IExampleItem): void => {
-    alert('item invoked: ' + item.name);
-  };
-
-  private _onSelectionChanged = (): void => {
-    if (this._hasMounted) {
-      this.forceUpdate();
-    }
-  };
+export const SelectionBasicExample: React.FunctionComponent = () => {
+  const [canSelect, setCanSelect] = React.useState<'all' | 'vowels'>('all');
+  const forceUpdate = useForceUpdate();
+  const items = useConst<IExampleItem[]>(() => createListItems(ITEM_COUNT));
+  const [selection, setSelection] = React.useState<Selection>(
+    () =>
+      new Selection({
+        onSelectionChanged: forceUpdate,
+        items,
+      }),
+  );
 
-  private _onToggleSelectAll = (): void => {
-    const { selection } = this.state;
+  const onToggleSelectAll = React.useCallback((): void => {
     selection.toggleAllSelected();
-  };
+  }, [selection]);
 
-  private _onSelectionModeChanged = (ev: React.MouseEvent<HTMLElement>, menuItem: IContextualMenuItem): void => {
-    this.setState((previousState: ISelectionBasicExampleState) => {
+  const onSelectionModeChanged = React.useCallback(
+    (ev: React.MouseEvent<HTMLElement>, menuItem: IContextualMenuItem): void => {
       const newSelection = new Selection({
-        onSelectionChanged: this._onSelectionChanged,
-        canSelectItem: previousState.canSelect === 'vowels' ? this._canSelectItem : undefined,
+        canSelectItem: canSelect === 'vowels' ? canSelectItem : undefined,
         selectionMode: menuItem.data,
+        onSelectionChanged: forceUpdate,
       });
-      newSelection.setItems(previousState.items, false);
-
-      return {
-        selection: newSelection,
-      };
-    });
-  };
-
-  private _onCanSelectChanged = (ev: React.MouseEvent<HTMLElement>, menuItem: IContextualMenuItem): void => {
-    const canSelectItem = menuItem.data === 'vowels' ? this._canSelectItem : undefined;
+      newSelection.setItems(items, false);
+      setSelection(newSelection);
+    },
+    [canSelect, forceUpdate, items],
+  );
 
-    this.setState((previousState: ISelectionBasicExampleState) => {
+  const onCanSelectChanged = React.useCallback(
+    (ev: React.MouseEvent<HTMLElement>, menuItem: IContextualMenuItem): void => {
       const newSelection = new Selection({
-        onSelectionChanged: this._onSelectionChanged,
-        canSelectItem: canSelectItem,
-        selectionMode: previousState.selection.mode,
+        canSelectItem: menuItem.data === 'vowels' ? canSelectItem : undefined,
+        selectionMode: selection.mode,
+        onSelectionChanged: forceUpdate,
+        items,
       });
-      newSelection.setItems(previousState.items, false);
-      return {
-        selection: newSelection,
-        canSelect: menuItem.data === 'vowels' ? 'vowels' : 'all',
-      };
-    });
-  };
-
-  private _canSelectItem = (item: IExampleItem): boolean => {
-    return /^[aeiou]/.test(item.name || '');
-  };
+      newSelection.setItems(items, false);
```

Comment:
You can remove this now that items are being set when the selection is constructed